src/object/cli_obj.c

```diff
@@ -1272,14 +1273,14 @@ dc_obj_open(tse_task_t *task)
 	if (rc != 0)
 		D_GOTO(out, rc);
 
-	rc = obj_ptr2pm_ver(obj, &obj->cob_md.omd_ver);
-	if (rc)
-		D_GOTO(out, rc);
-
 	rc = obj_layout_create(obj, false);
 	if (rc != 0)
 		D_GOTO(out, rc);
 
+	rc = obj_ptr2pm_ver(obj, &obj->cob_md.omd_ver);
+	if (rc)
+		D_GOTO(out, rc);
```

Comment:
since in obj_layout_create it assigns obj->cob_md.omd_ver, seems here need not do obj_ptr2pm_ver again?

Reply:
yes, probably not, I am not sure what is the original reason why put it there, so I just leave it there.